src/muya/lib/eventHandler/keyboard.js

```diff
@@ -66,10 +66,6 @@ class Keyboard {
       if (event.target.closest('[contenteditable=false]')) {
         return
       }
-      const { start, end } = selection.getCursorRange()
```

Comment:
I think we still need this because the `selectionChange: expected cursor but cursor is null.` exceptions occur again. If currently is no cursor available the cursor will not be set in the next "tick"/after this function.

Reply:
My fault, I thought this code was useless, so I deleted it.